src/carplay.cs

```diff
@@ -1008,4 +1289,556 @@ interface CPTemplateApplicationDashboardScene {
+		CGSize MaximumImageSize { get; }
+
+		[Field ("CPMaximumNumberOfGridImages")]
+		nuint MaximumNumberOfGridImages { get; }
+
+		[NullAllowed, Export ("text")]
+		string Text { get; set; }
+
+		[NullAllowed, Export ("userInfo", ArgumentSemantic.Strong)]
+		NSObject UserInfo { get; set; }
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (14,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface CPTextButton
+	{
+		[Export ("initWithTitle:textStyle:handler:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (string title, CPTextButtonStyle textStyle, [NullAllowed] Action<CPTextButton> handler);
+
+		[Export ("title")]
+		string Title { get; set; }
+
+		[Export ("textStyle", ArgumentSemantic.Assign)]
+		CPTextButtonStyle TextStyle { get; set; }
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (14,0)]
+	[BaseType (typeof (CPBarButton))]
+	interface CPMessageComposeBarButton
+	{
+		[Static]
+		[Export ("new")]
+		[return: Release]
+		CPMessageComposeBarButton Create ();
+
+		[Export ("initWithImage:")]
+		IntPtr Constructor (UIImage image);
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (14,0)]
+	[BaseType (typeof (NSObject))]
+	interface CPMessageListItem : CPListTemplateItem
+	{
+		[Internal]
+		[Export ("initWithConversationIdentifier:text:leadingConfiguration:trailingConfiguration:detailText:trailingText:")]
+		IntPtr InitWithConversationIdentifier (string conversationIdentifier, string text, CPMessageListItemLeadingConfiguration leadingConfiguration, [NullAllowed] CPMessageListItemTrailingConfiguration trailingConfiguration, [NullAllowed] string detailText, [NullAllowed] string trailingText);
+
+		[Internal]
+		[Export ("initWithFullName:phoneOrEmailAddress:leadingConfiguration:trailingConfiguration:detailText:trailingText:")]
+		IntPtr InitWithFullName (string fullName, string phoneOrEmailAddress, CPMessageListItemLeadingConfiguration leadingConfiguration, [NullAllowed] CPMessageListItemTrailingConfiguration trailingConfiguration, [NullAllowed] string detailText, [NullAllowed] string trailingText);
+
+		[NullAllowed, Export ("conversationIdentifier")]
+		string ConversationIdentifier { get; set; }
+
+		[NullAllowed, Export ("phoneOrEmailAddress")]
+		string PhoneOrEmailAddress { get; set; }
+
+		[Export ("leadingConfiguration", ArgumentSemantic.Strong)]
+		CPMessageListItemLeadingConfiguration LeadingConfiguration { get; set; }
+
+		[NullAllowed, Export ("trailingConfiguration", ArgumentSemantic.Strong)]
+		CPMessageListItemTrailingConfiguration TrailingConfiguration { get; set; }
+
+		[NullAllowed, Export ("detailText")]
+		string DetailText { get; set; }
+
+		[NullAllowed, Export ("trailingText")]
+		string TrailingText { get; set; }
+
+		[Field ("CPMaximumMessageItemImageSize")]
+		CGSize MaximumMessageItemImageSize { get; }
+
+		[NullAllowed, Export ("text")]
+		string Text { get; set; }
+
+		[NullAllowed, Export ("userInfo", ArgumentSemantic.Strong)]
+		NSObject UserInfo { get; set; }
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (14,0)]
+	[BaseType (typeof (NSObject))]
```

Comment:
`[DisableDefaultCtor]` as the content can't be edited and default values likely don't make sense